ext_mongodb.php

```diff
@@ -853,21 +1100,51 @@ public function __debugInfo() : array
 
 final class UTCDateTime implements Type, \Serializable
 {
-	use DenySerialization;
-
 	private int $milliseconds;
 
+	static private function checkArray( array $state )
+	{
+		if (
+			!array_key_exists( 'milliseconds', $state ) ||
+			! ( is_int( $state['milliseconds'] ) || is_string( $state['milliseconds'] ) )
+		) {
+			throw new \MongoDB\Driver\Exception\InvalidArgumentException( "MongoDB\BSON\UTCDateTime initialization requires \"milliseconds\" integer or numeric string field" );
+		}
+	}
+
+	public function serialize() : string
+	{
+		return serialize( [
+			'milliseconds' => (string) $this->milliseconds,
```

Comment:
IMO, we should always use strings for serialization, as to be the same in each context. I did not have PHPC-726 in mind.